mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -242,7 +275,12 @@ class GeneralizedLinearRegression @Since("2.0.0") (@Since("2.0.0") override val
   def setLinkPredictionCol(value: String): this.type = set(linkPredictionCol, value)
 
   override protected def train(dataset: Dataset[_]): GeneralizedLinearRegressionModel = {
-    val familyObj = Family.fromName($(family))
+    val familyObj = if ($(family) == "tweedie") {
+      new Tweedie($(varPower))
```

Comment:
`Family` is not subclass of `GeneralizedLinearRegression`. Could you elaborate how to make it get the `variancePower` value?